modules/ingest-common/license/log4j-api-LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
the license file should not be needed since the log4j jar is not actually bundled in this module; it is part of server

Reply:
fixed. thanks